main/zoe/guide/contracts/callSpread.md

```diff
@@ -0,0 +1,77 @@
+# Call Spread
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/callSpread.js) (Last updated: 03-NOV-2020)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+This contract implements a fully collateralized call spread. A call spread can be used as a
+financial building block to create futures, puts, calls, and event binaries that would form the
+basis for a prediction market, insurance, and much more. A Call Spread is a combination of a call
```

Comment:
Should be consistent on "Call Spread" capitalization; previously on line 8 you had "call spread".